inc/TRestRawDreamToSignalProcess.h

```diff
@@ -0,0 +1,198 @@
+#include <TH1D.h>
+#include <TH2D.h>
+#include <TMath.h>
+#include <TROOT.h>
+#include <TSystem.h>
+#include <TTree.h>
+
+#include <arpa/inet.h>
+#include <iostream>
+#include <list>
+#include <map>
+#include <string>
+#include <vector>
+#include "math.h"
+
+//! An process to read binary data from FEUDream electronics
+class TRestRawDreamToSignalProcess : public TRestRawToSignalProcess {
+   protected:
+    class DataLineDream {
+       public:
+        DataLineDream() { data = 0; }
+        ~DataLineDream() {}
+        void ntohs_() { data = ntohs(data); };
+        bool is_final_trailer() const { return (((data)&0x7000) >> 12) == 7; }   // X111
+        bool is_end_of_event() const { return (((data)&0x7800) >> 11) == 0xF; }  // X1111
+        bool is_data_trailer() const { return (((data)&0x6000) >> 13) == 2; }    // X10X
+        bool is_first_line() const { return (((data)&0x7000) >> 12) == 3; }      // X011
+        bool is_data() const { return (((data)&0x7000) >> 12) == 0; }            // X000
+        bool is_data_zs() const { return (((data)&0x6000) >> 13) == 0; }         // X00X
+        bool is_channel_ID() const { return (((data)&0x7000) >> 12) == 1; }      // X001
+        bool is_Feu_header() const { return (((data)&0x7000) >> 12) == 6; }      // X110
+        bool is_data_header() const { return (((data)&0x6000) >> 13) == 1; }     // X01X
+        bool get_zs_mode() const { return (((data)&0x400) >> 10); }
+        int get_Feu_ID() const { return (((data)&0xFF)); }
+        long int get_finetstp() const { return (((data)&0x0007)); }
+        //#define GET_FINETSTP(word)      (word & 0x0007)
+        int get_sample_ID() const { return (((data)&0xFF8) >> 3); }
+        int get_channel_ID() const { return (((data)&0x3F)); }
+        int get_dream_ID() const { return (((data)&0xE00) >> 9); }  // non-zS mode
+        int get_dream_iD() const { return (((data)&0xE00) >> 6); }  // for zS mode
+        int get_TimeStamp_Op() const { return (((data)&0x1FF)); }
+        int get_data() const { return (((data)&0xFFF)); }
+        unsigned short int data;
+    };
+
+    class FeuReadOut {
+       public:
+        int Id;
+        int N;
+        DataLineDream current_data;
+        bool data_to_treat;
+        bool event_completed;
+        bool last_event;
+        bool FeuHeaderLoaded;
+        int FeuHeaderLine;
+        int DataHeaderLine;
+        int DataTrailerLine;
+        int asicN;
+        int channelN;
+        int channel_data;
+        int physChannel;
+        int EventID;
+        int EventID_Op;
+        long long int TriggerID;
+        int TriggerID_ISB;
+        int TriggerID_LSB;
+        int CMN;
+        int CMN_rest;
+        long int Cell_ID;
+        int Cell_ID_ISB;
+        int Cell_ID_LSB;
+        int Cell_ID_MSB;
+        long long int TimeStamp;
+        long int TimeStamp_Op1;
+        long int TimeStamp_Op2;
+        long int TimeStamp_Op3;
+        long int FineTimeStamp;
+        int isample;
```

Comment:
Same comment as before, please consider to move to `uint8_t`, `uint16_t`, `uint32_t` or `uint64_t` instead.

Reply:
Ok, it happened, changing `int` to `uint32_t`, `short int` to `uint16_t` and `long int` to `uint64_t` provoked the validation to fail.

When I reverted the change, the pipeline succeeded again.

https://gitlab.cern.ch/rest-for-physics/rawlib/-/commit/bc9b1614369914d6c81dec76ae96c12e3629e398

It must be a delicate change...

Reply:
Perhaps the problem is the signature? But in general shouldn't be an issue, I did similar changes in #12 and I didn't saw any issue, but I don't know if there is validation...

Can you just create an issue about this to be followed up?